desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -7,7 +7,7 @@ openBuyersPremiumModal = require './components/buyers_premium/index.coffee'
 AuthModalView = require '../../../../components/auth_modal/view.coffee'
 inquire = require '../../lib/inquire.coffee'
 acquire = require '../../lib/acquire.coffee'
-helpers = require './helpers.coffee'
+ViewHelpers = require './helpers.coffee'
```

Comment:
I'm open to either. in one component [main_layout i believe] there is some logic around an already existing `helpers`, which i think is why in artwork/auction/templates/live.jade above the `helpers` gets namespaced. Then in my_active_bids there is a ViewHelpers, which I spent a while trying to track down before @sweir27 showed me the way. I intended to be consistent but looks like I still missed some.